spec/requests/generic_object_definitions_spec.rb

```diff
@@ -0,0 +1,286 @@
+# rubocop:disable Style/WordArray
```

Comment:
Disabling as I prefer that the sample requests not use word arrays to better align with how the request will look.